refinery/user_files_manager/tests.py

```diff
@@ -1,16 +1,43 @@
 import logging
+import requests
 
-from django.test import TestCase
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from rest_framework.test import (APIRequestFactory,
+                                 APITestCase, force_authenticate)
+from urlparse import urljoin
 
+from core.models import User
+from views import UserFiles
 
 logger = logging.getLogger(__name__)
 
 
-class UserFilesAPITests(TestCase):
-    pass
-    # TODO: Fails, because solr isn't running.
-    # Should I mock it, or make this an integration test?
+# def fake_search_solr():
+#     return {}
+#
+# @mock.patch("user_files_manager.utils.search_solr",
```

Comment:
I'd remove comments, but I'm actually more interested if you are able to mock a function call for the entirety of a test suite, this would be really useful for me.